server/models/BookingRequest.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require("mongoose");
+
+const bookingRequestSchema = new mongoose.Schema({
+  user_id: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  sitter_id: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  start: {
+    type: Date,
+    default: Date.now,
+    required: true,
+  },
+  end: {
+    type: Date,
+    default: Date.now,
+    required: true,
+  },
+  description: {
+    type: String,
+    default: "",
+  },
+  accepted: {
+    type: Boolean,
+    default: false,
+  },
+  declined: {
+    type: Boolean,
+    default: false,
+  },
+  paid: {
+    type: Boolean,
+    default: false,
+  },
```

Comment:
It's good to also set mongoose `timestamps` option to `true` to automatically manage `createdAt` and `updatedAt` properties.

Reply:
resolved by 18e0a1def88ab81284aa3b55fd56b64437e7d490